frontend/src/component/common/UserAvatar/UserAvatar.tsx

```diff
@@ -85,4 +97,26 @@ export const UserAvatar: FC<IUserAvatarProps> = ({
             />
         </StyledAvatar>
     );
+
+    const tooltip = disableTooltip ? undefined : tooltipContent(user);
+    if (tooltip) {
```

Comment:
How about simpler
```suggestion
    if (!disableTooltip) {
      const tooltip = tooltipContent(user);
```

Reply:
Or even more plain, with early return `if (disableTooltip) {return Avatar}`

Reply:
Well, yes and no. Even if we have a user, there's no guarantee that it has anything we can put into a tooltip. It's defined as 
```tsx
    user?: Partial<
        Pick<IUser, 'id' | 'name' | 'email' | 'username' | 'imageUrl'>
    >;
```

which means that `{}` is also valid, right?

So we can totally do an early return, but we'd also have to check for it later. Another option would be to extract the tooltip into a separate component that returns either a tooltip with the avatar as child or `<>{children}</>` if we don't know how to make a tooltip.

Reply:
Tried a few different ways, but ran into some TS typing errors. I don't want to spend more time debugging that for now, so I'll leave it for now. We can change it later if we want to.